### PR TITLE
chore: Adds release-please workflow.

### PR DESCRIPTION
**Pull request by yorik**
# What ❔

Adds release-please workflow triggered by push into main branch.

## Why ❔

Preparing to use the repo for releases.

## Checklist

- [x] PR title corresponds to the body of PR (we generate changelog entries from PRs).
- [ ] Tests for the changes have been added / updated.
- [ ] Documentation comments have been added / updated.
- [ ] Code has been formatted via `zk fmt` and `zk lint`.

### CODE REVIEW

**Review by shahar4 — APPROVED** [review]

File: .github/release-please/config.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+  "separate-pull-requests": true,
+  "group-pull-request-title-pattern": "chore: release ${component} ${branch}",
+  "bootstrap-sha": "a980ac61f484a3b6bdbe11e606977d0d92d4e9ff",
```
Comment (Deniallugo) [inline .github/release-please/config.json:4]: Don't forget to point it to the latest commit before merging

Comment (shahar4) [reply .github/release-please/config.json:4]: Will do

Comment (yorik) [reply .github/release-please/config.json:4]: I've already did.

**Review by Deniallugo — APPROVED** [review]

File: .github/release-please/manifest.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+  "sdk/zksync-web3.js": "0.15.4",
+  "sdk/zksync-rs": "0.4.0",
+  "core": "15.0.1",
+  "prover": "7.1.1"
+}
```
[the remark this answers is not in the document]
Comment (yorik) [reply .github/release-please/manifest.json:6]: ack

**Pull request merged**